### DISCUSSION TITLE
Installing via Windows Service

### DISCUSSION THREAD

**Discussion by mkshenoy** [rogerfar/rdt-client#518] [Q&A]
I configured the appsettings.json with a logging path and db path as in the installation instruction and ran service-install.bat. The service gets created, but doesn't run.  If i try to start the service manually i get this error "error 1053 the service did not respond to the start or control request in a timely fashion"
Could Anyone help me if i am doing anything wrong?

Thanks in advance

**Comment by rogerfar** [comment]
Check your event viewer, most likely you are missing the correct .NET runtime.
